src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -184,5 +185,51 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
       val actualOutput = salutation.asInstanceOf[WdlString].value.trim
       actualOutput should endWith("/call-whereami")
     }
+
+    "build metadata correctly" in {
+
+      implicit val workflowManagerActor = TestActorRef(WorkflowManagerActor.props(CromwellSpec.BackendInstance), self, "Test Workflow metadata construction")
+
+      val workflowId = waitForHandledMessagePattern(pattern = "transitioning from Running to Succeeded") {
+        messageAndWait[WorkflowId](SubmitWorkflow(new SampleWdl.ScatterWdl().asWorkflowSources()))
+      }
+
+      val status = messageAndWait[Option[WorkflowState]](WorkflowStatus(workflowId)).get
+      status shouldEqual WorkflowSucceeded
+
+      val metadata = messageAndWait[WorkflowMetadataResponse](WorkflowMetadata(workflowId))
+      metadata should not be null
+
+      metadata.status shouldBe WorkflowSucceeded.toString
+      metadata.start shouldBe defined
+      metadata.end shouldBe defined
+      metadata.outputs shouldBe defined
+      metadata.outputs.get should have size 5
+      metadata.calls should have size 5
+
+      // ok if this explodes, it's a test
+      val devOutputs = metadata.outputs.get.get("w.A.A_out").get
+      val wdlArray = devOutputs.asInstanceOf[WdlArray]
+      wdlArray.wdlType shouldBe WdlArrayType(WdlStringType)
+
+      (wdlArray.value map { case WdlString(string) => string }) shouldEqual Vector("jeff", "chris", "miguel", "thibault", "khalid", "scott")
+
+      val devCalls = metadata.calls.get("w.C").get
+      devCalls should have size 6
+      devCalls foreach { call =>
+        call.start shouldBe defined
+        call.end shouldBe defined
+        call.jobId should not be defined
+        call.returnCode.get shouldBe 0
+        call.stdout shouldBe defined
+        call.stderr shouldBe defined
+        call.inputs should have size 1
+        call.backend.get shouldEqual "Local"
+        call.backendStatus should not be defined
+        call.executionStatus shouldBe "Done"
+      }
+
+      (devCalls map { _.outputs.get.get("C_out").get.asInstanceOf[WdlInteger].value } ) shouldEqual Vector(400, 500, 600, 800, 600, 500)
```

Comment:
When I hit this with an IntelliJ formatting I get `})`, so I'll change to that.  I also get an expression smeared out over 3 lines, which I will not change to.  :smile: